### PR TITLE
Cartoons - About us page

### LINKED ISSUE #21: Static Page: Cartoons - About us
Redevelop https://mathsoc.uwaterloo.ca/mathsoc-cartoons-main-page/ for the new website.

**Acceptance criteria:** The page is rebuilt, containing all information from the version on the old site.

### PR DESCRIPTION
**Pull request by ad-world**
This should close #21 

It creates the page, styles, and data for the cartoons about us page

Test it locally by going to the ```cartoons-page``` branch. You can find the page under Cartoons -> About us on the navbar, or at ```localhost:3000/cartoons/about-us```

### CODE REVIEW

File: views/pages/cartoons/about-us.pug [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                    h3 #{data.joinUs}
+                    p(class="app-caption") #{data.applicationsCaption}
+                    div!=data.signupMarkup
+                    div(style="width: 100%; text-align:center;")
+                        +pink-button(data.buttons.team.text, '', data.buttons.team.link)
+                        br
+                        +pink-button(data.buttons.archive.text, '', data.buttons.archive.link)
+                div(class="col")
+                    - var images = data.carouselImages;
+                    +carousel(data.carouselImages)
```
Comment (some-molasses) [inline views/pages/cartoons/about-us.pug:34]: looks like the negative margin used to shrink the homepage carousel is now cutting off the top of the image on the cartoons carousel.

image

**Review by some-molasses — CHANGES_REQUESTED** [review]

File: views/pages/cartoons/about-us.pug [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            div(class="title get-in-touch")
+                h1 #{data.getInTouch}
+            div(class="row button-row")
```
[the remark this answers is not in the document]
Comment (ad-world) [reply views/pages/cartoons/about-us.pug:37]: fixed

File: views/pages/cartoons/about-us.pug (hunk 1 above)
Comment (ad-world) [reply views/pages/cartoons/about-us.pug:34]: fixed

**Review by some-molasses — COMMENTED** [review]
image

sorry for not catching it earlier but I just noticed that we can really improve on the previous site here!  can we:
 * use the cartoons image as the banner (and remove it from the body)
 * remove the 'about us' heading

again sorry for the double change request 💀

**Review by some-molasses — APPROVED** [review]
looks good!

**Pull request merged**